### PR TITLE
prepare umd build before publish

### PR DESCRIPTION
**Pull request by just-boris**
* added browserify build on prepublish hook
* added example how to use UMD build
* updated package.json to exclude unnecessary files from npm package (examples and test)

### CODE REVIEW

**Comment by just-boris** [comment]
fixes #105

File: package.json [hunk 1]
```diff
@@ -10,7 +10,11 @@
     "test:browser": "standard && npm run deps && NODE_ENV=test zuul tests/browser/*",
     "test:browser:local": "standard && npm run deps && NODE_ENV=test zuul --local 8080 -- tests/browser/*",
     "test:cov": "npm run test:server:cov && npm run test:browser",
-    "test": "npm run test:server && npm run test:browser"
+    "test": "npm run test:server && npm run test:browser",
+    "start": "http-server",
```
Comment (yoshuawuyts) [inline package.json]: I reckon this script should best be scoped to `examples/vanilla`

**Comment by yoshuawuyts** [comment]
Wooh, thanks! - Just left one comment; rest looks brilliant to me! :sparkles:

File: package.json (hunk 1 above)
Comment (just-boris) [inline package.json]: It uses files from `dist` directory. So we need to serve from project root.

Comment (yoshuawuyts) [inline package.json]: A relative path can be passed as the first argument to `http-server` to determine the location served:
```sh
$ http-server ../../
```

**Comment by just-boris** [comment]
I realised that is it possible to open the example, as any local html-file with `file://` protocol.
No need to keep http-server at all

**Comment by yoshuawuyts** [comment]
Could you remove the `.gitkeep` file?

**Comment by just-boris** [comment]
@yoshuawuyts now we are writing output file using standard `>` operator.
`.gitkeep` is used to create this directory after `git clone`, because `>` fails if target directory doesn't exist.

Do you know another solutions how to automatically create the `dist` folder if it doesn't exist?

**Comment by yoshuawuyts** [comment]
Gehe, unix tools to the rescue!
```sh
$ mkdir -p 'dist/'
```

**Comment by just-boris** [comment]
Done.

Added this as first step of `prepublish` script.

**Comment by timwis** [comment]
This is awesome! One thing as a heads up: 3.0 separates `choo.view` into `require('choo/html')`, a separate file from `index.js`, so I imagine this script may need to be updated to publish both when that's ready.

**Comment by yoshuawuyts** [comment]
@timwis ooh; good call hey

**Comment by yoshuawuyts** [comment]
I've merged the commits into #159 and rebased onto master to resolve the merge conflicts. Heaps thanks!

**Comment by just-boris** [comment]
Good. Anyway I was glad to help you :+1:

**Pull request closed without merging**